notebooks/.devcontainer/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.9.19
+FROM python:3.11.10
 # create non-root user and set the default user
 ARG USERNAME=devuser
-ARG USER_UID=1000
+ARG USER_UID=1010
```

Comment:
just for my understanding, what is the need to update this UID?

Reply:
gpu side's uuid was updated to 1010 due to the reason commented in dockerfile. To keep the file permission to the same user across two containers, I needed to update uuid on this container to the same id with gpu's one

```
# uid=1000 is already used for ubuntu so use 1010 instead
ARG USER_UID=1010
```